nightingale_dispatcher/src/nightingale_dispatcher/navigate_task.py

```diff
@@ -1,13 +1,14 @@
 #!/usr/bin/env python3
 
 import rospy
-from nightingale_dispatcher.task import Task
+
+from nightingale_dispatcher.task_with_interface_comms import TaskWithInterfaceComms
 import actionlib
 from actionlib_msgs.msg import GoalStatus
 from nightingale_msgs.msg import RoomRunnerAction, RoomRunnerGoal
 
 
-class NavigateTask(Task):
+class NavigateTask(TaskWithInterfaceComms):
```

Comment:
The NavigateTask doesn't use the interface, it can inherit directly from Task

Reply:
I was thinking it would need to change the interface screen to the driving screen and would need to then inherit from the task with interface. Otherwise we would have to update the tablet that we are driving in a different task